lib/event-listeners/food-listener.js

```diff
@@ -85,12 +90,62 @@ const getFilterTermRegExp = () => {
 }
 
 const prepareTableForFilter = () => {
-    $("tr").hide()
+    $("table.searchable tr").hide()
     $("th").parents().show()
 }
 
 const filterFoods = (regexTerm) => {
-    $("tr").filter(function() {
+    $("table.searchable tr").filter(function() {
         return regexTerm.test($(this).children().text());
     }).show()
 }
+
+// sort - might need to be in a class
+$(document).on({
+    click: function() {
+        let table = $(this).parents('table')
+        sortFoodTable(table)
+    }
+}, '#calorie-header')
+
+const sortTableAscending = (table) => {
+    let sortableRows = table.children('tr')
+    let sortedRows = sortableRows.sort((a, b) => {
+        let valueA = parseInt($(a).children('td.calories').text())
+        let valueB = parseInt($(b).children('td.calories').text())
+        return (valueA < valueB) ? -1 : (valueA > valueB) ? 1 : 0;
+    })
+    sortableRows.remove()
+    $(table).append(sortedRows).addClass('asc').removeClass('alphabetical')
+}
+
+const sortTableDescending = (table) => {
+    let sortableRows = table.children('tr')
+    let sortedRows = sortableRows.sort((a, b) => {
+        let valueA = parseInt($(a).children('td.calories').text())
+        let valueB = parseInt($(b).children('td.calories').text())
+        return (valueA > valueB) ? -1 : (valueA < valueB) ? 1 : 0;
+    })
+    sortableRows.remove()
+    $(table).append(sortedRows).addClass('desc').removeClass('asc')
+}
+
+function defaultSort(table) {
+    let sortableRows = table.children('tr')
+    let sortedRows = sortableRows.sort((a, b) => {
+        let valueA = $(a).children('td.name').text()
+        let valueB = $(b).children('td.name').text()
+        return (valueA.toUpperCase() < valueB.toUpperCase()) ? -1 : (valueA.toUpperCase() > valueB.toUpperCase()) ? 1 : 0;
+    })
+    sortableRows.remove()
+    $(table).append(sortedRows).addClass('alphabetical').removeClass('desc')
+}
+
+const sortFoodTable = (table) => {
+  if (table.hasClass('alphabetical'))
+    return sortTableAscending(table)
+  else if (table.hasClass('asc'))
+    return sortTableDescending(table)
+  else if (table.hasClass('desc'))
+    return defaultSort(table)
+}
```

Comment:
@kat3kasper we have the food tables alphabetizing by default, now. It is not in the user stories, but we felt it was best from a ux standpoint. We can remove if needed. We are planning a refactor to pull functionality out into classes on most of our files, too.